sklearnex/tests/test_memory_usage.py

```diff
@@ -109,7 +109,7 @@ def remove_duplicated_estimators(estimators_list):
 )
 estimators = [
     PreviewPCA,
-    PreviewLinearRegression,
+    LinearRegression,
```

Comment:
All non-preview patched estimators are added by default
```suggestion
```

Reply:
Thanks for clarification